### ISSUE TITLE
[Not Planned] AWS Distribution for Kubeflow-1.8

### ISSUE THREAD

**Issue by surajkota** [awslabs/kubeflow-manifests#794]
AWS is not planning to participate in distribution testing for 1.8 release at this time. If anything changes in the future, we will let you know. Please reach out to us via [Github issue](https://github.com/awslabs/kubeflow-manifests/issues/794) if you have any questions.

**Comment by pragadeeshraju** [comment]
Hello @surajkota, I'd like to verify if there are any plans for AWS to discontinue future releases/support of Kubeflow on AWS.?

**Comment by sachdevayash1910** [comment]
Will there be a simple way to integrate the  KFP v2 GA release with KF 1.7 then? Once it is released? If we are able to do this seamlessly, it will really helpful

**Comment by ghaering** [comment]
> Hello @surajkota, I'd like to verify if there are any plans for AWS to discontinue future releases/support of Kubeflow on AWS.?

Yes, this would be really interesting. It would mean we will have to maintain our fork independently.

**Comment by kevdougful** [comment]
Agreed. This would significantly increase our ongoing development efforts. Some clarity on future support/development would be greatly appreciated. Does "at this time" mean "sometime later" or "not ever?" 😄

**Comment by RusLin-oir** [comment]
hoping that the closing comments of this issue can be published somewhere so that we can keep track of where this is heading.

**Comment by nadella45** [comment]
@surajkota hoping to see the aws build corresponding to Kubeflow 1.8 soon. Request to consider for a early release.

**Comment by thesuperzapper** [comment]
While I hope that AWS continues first-party support for Kubeflow, I would love to share an alternative called [__deployKF__](https://www.deploykf.org/).

deployKF is a community-maintained Kubeflow distribution that takes a different approach from the others. It is much easier to maintain because it uses “values” (like a helm chart), rather than manual YAML patches. This approach enables features like in-place upgrades, and fully declarative configs (for GitOps) while also being way easier to [connect with external services like S3](https://www.deploykf.org/guides/tools/external-object-store/) and [RDS](https://www.deploykf.org/guides/tools/external-mysql/).

The next version of deployKF will support Kubefow 1.8, and current deployKF users will be able to do an in-place upgrade!

Also, we are about to publish a "deployKF on AWS" guide, but if you are already familiar with AWS and EKS, you don't need to wait, just follow the generic [getting started](https://www.deploykf.org/guides/getting-started/) guide.

---

_If you want to help deployKF, you can participate on the [`deploykf/deploykf`](https://github.com/deployKF/deployKF) repo, or just give the repo a ⭐ on GitHub!_

_PS: deployKF is actually [planning to support more than Kubeflow](https://www.deploykf.org/reference/future-tools/) in the near future!_

**Comment by LJArendse** [comment]
@surajkota Any feedback regarding:
* Kubeflow 1.8 support?
* _Kubeflow on AWS_ repository future support/development?

**Comment by hfarooqui** [comment]
Wish there was a solid alternative to "Kubeflow on AWS" :/

**Comment by thesuperzapper** [comment]
> Wish there was a solid alternative to "Kubeflow on AWS" :/

@hfarooqui does [deployKF](https://www.deploykf.org/) meet your needs?

If not, I would love to get your feedback so that we can make it work for your use case.

**Comment by celha** [comment]
Did someone find a workaround for using kfp v2 for kubeflow on aws?
